tests/ctst/steps/utils/utils.ts

```diff
@@ -197,17 +197,18 @@ async function createBucketWithConfiguration(
 
 async function putObject(world: Zenko, objectName?: string) {
     world.addToSaved('objectName', objectName || Utils.randomString());
-    const objectNameArray = world.getSaved<string[]>('objectNameArray') || [];
-    objectNameArray.push(world.getSaved<string>('objectName'));
-    world.addToSaved('objectNameArray', objectNameArray);
     await uploadSetup(world, 'PutObject');
     world.addCommandParameter({ key: world.getSaved<string>('objectName') });
     world.addCommandParameter({ bucket: world.getSaved<string>('bucketName') });
     const result = await S3.putObject(world.getCommandParameters());
+    const versionId = extractPropertyFromResults<string>(result, 'VersionId');
+    world.saveCreatedObject(world.getSaved<string>('objectName'), versionId || '');
+
     world.addToSaved('versionId', extractPropertyFromResults(
```

Comment:
saving `versionId` seems redundant with `world.saveCreatedObject` ; even if we keep it to avoid too wide-ranging changes to the tests, should it not be done inside `saveCreatedObject` ?

Reply:
I removed the lines, and updated the function: it was storing always the last version id, but we may need to store all the known version IDs...

Reply:
-> ZENKO-4904